lib/commands/waitForExist.js

```diff
@@ -65,7 +65,9 @@ module.exports = function waitForExist(selector, ms, reverse) {
 
                 var now = new Date().getTime();
                 if(now - start > ms) {
-                    return cb(null, false);
```

Comment:
I need to redo this again. Think about: if the element wasn't found then it doesn't exist and that's the reason why we are waiting, so we should return true or false:

```js
if(now - start > ms) {
    return cb(null, reverse);
}